molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/resources/Resources.java

```diff
@@ -24,4 +27,11 @@
 	 */
 	Iterable<Entity> findAll(String name, Query q);
 
+	/**
```

Comment:
To be honest I don't know mark did some work on this too might be his code. But I suspect that it has something to do with more then one hit found in the resource files.